examples/azure-identity/go/Makefile

```
@@ -0,0 +1,64 @@
+REGISTRY ?= ghcr.io/azure/azure-workload-identity
+IMAGE_NAME := azid-go
+IMAGE_VERSION ?= latest
+
+DEMO_IMAGE := $(REGISTRY)/$(IMAGE_NAME):$(IMAGE_VERSION)
+
+## --------------------------------------
+## Images
+## --------------------------------------
+
+# Output type of docker buildx build
+OUTPUT_TYPE ?= type=registry
+
+ALL_OS = linux windows
+ALL_ARCH.linux = amd64 arm64
+ALL_ARCH.windows = amd64
+ALL_OSVERSIONS.windows := 1809 1903 1909 2004 ltsc2022
```

```suggestion
ALL_OSVERSIONS.windows := 1809 ltsc2022
```

i think rest are eol?

---

Confirmed those versions are EOL:
- https://learn.microsoft.com/en-us/lifecycle/announcements/windows-10-1903-end-of-servicing
- https://learn.microsoft.com/en-us/lifecycle/announcements/windows-10-1909-end-of-servicing
- https://learn.microsoft.com/en-us/lifecycle/announcements/windows-10-version-2004-end-of-servicing